data/human/transport missions.txt

```diff
@@ -3582,3 +3582,732 @@ mission "Hauler VI cargo prototype 3"
 			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
 			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
 			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`
+
+
+
+
+	name "Take family home to <planet>"
+	description "You meet a family worried about the potential destruction of war in the Free Worlds and want to cut their vacation short and go home to <destination>."
+	source
+		government "Free Worlds"
+		attributes "tourism"
+	destination "Mainsail"
+	passengers 3
+	cargo "personal belongings" 1
+	to offer
+		has "FW Katya 7C: done"
+	to fail
+		has "free worlds plot completed"
+	on offer
+		conversation
+			`After walking around the spaceport for awhile you head back to your ship to find and middle aged couple with their two grade school daughters waiting for you at your hanger. The man is busy trying to adjudicate a dispute between the girls but the lady notices you immediately and begins to approach you. "Thank goodness you're here," she says. "We were hoping you could get us back home to the paradise planets. This war caught us right in the middle of our vacation and we can't risk being anywhere near here when the Navy attacks, especially with our little 'angels'," she gestures toward her daughters, now in a full blown fight. "If you can find the time to take us we will pay you very well."`
+			choice
+				`	"Sure, I'll take you."`
+				`	"OK. But it's pretty far. It better be worth it."`
+				`	"No. I don't think so. Sorry."`
+					decline
+			`	"Then it's a deal," she shakes your hand firmly. The whole family gets onto the <ship> without another word, leaving you to load their many bags and cases.`
```

Comment:
`	"Then it's a deal," she shakes your hand firmly. The whole family gets onto your ship without a word, leaving you to load their many bags and cases.`